registry/default/ui/profile-avatar-button.tsx

```diff
@@ -219,10 +219,15 @@ const ProfileDidField = ({ className }: React.ComponentProps<"div">) => {
   );
 };
 
-ProfileAvatarButton.Avatar = ProfileAvatar;
-
-ProfileAvatarButton.AvatarFallback = ProfileAvatarFallback;
-
-ProfileAvatarButton.AvatarImage = ProfileAvatarImage;
-
-export { ProfileAvatarButton, ProfileName, ProfileDidField };
```

Comment:
Why did you change this ? The whole idea is to avoid confusion with the ProfileAvatar component we have. That's what happens when it's simply re-exported within ProfileAvatarButton. 

But why you made the change ?